lib/private/Files/View.php

```diff
@@ -480,6 +482,7 @@ public function readfilePart($path, $from, $to) {
 						$len = $chunkSize;
 					}
 					echo fread($handle, $len);
+					$this->checkConnectionStatus();
 					flush();
```

Comment:
It should also be changed here:
```suggestion
					flush();
					$this->checkConnectionStatus();
```